src/main/java/org/p632/turnkey/beans/ProjectBuilderBean.java

```diff
@@ -20,24 +19,21 @@
 @Component
 public class ProjectBuilderBean {
 
-	
-	public List<String> GetDependencyList()
-	{
+	public List<String> getDependencyList() {
 		List<String> listDeps = new ArrayList<String>();
-		
-		try 
-		{
+
+		try {
 			ClassLoader classLoader = getClass().getClassLoader();
-			File xmlFile = new File(classLoader.getResource("dependency.xml").getFile());			
+			File xmlFile = new File(classLoader.getResource("dependency.xml").getFile());
 			DocumentBuilderFactory dbFactory = DocumentBuilderFactory.newInstance();
 			DocumentBuilder dBuilder = dbFactory.newDocumentBuilder();
-			Document doc = dBuilder.parse(xmlFile);		
+			Document doc = dBuilder.parse(xmlFile);
```

Comment:
In the future you might save yourselves some headaches by looking into something like https://github.com/FasterXML/jackson-dataformat-xml